### ISSUE TITLE
Disable DataFixerUpper Option Should not Exist

### ISSUE THREAD

**Issue by ryleu** [TheRandomLabs/RandomPatches#227]
Like the title says. The option is WAY too dangerous and it doesn't fix a big. People should instead use a mod such as LazyDFU, which only compiles DFU rules when they're needed (and reduces startup time massively), instead of disabling the DFU outright.

Sorry if this seems somewhat aggressive, but I don't want people to accidentally ruin their worlds because of an option they don't understand.

**Comment by K0-RR** [comment]
That's up to people if they ruin their world. The feature is disabled by default and enabling it is "not recommended". It is stated clearly that this feature is dangerous and users should not mess with options they don't understand. Maybe the mention of LazyDFU could be added as a recommendation there but I see no reason to remove this option entirely.

**Comment by ryleu** [comment]
> It is stated clearly that this feature is dangerous and users should not mess with options they don't understand. 

True, but why is it there at all? It isn't a bugfix in a mod called Random**Patches**.

> Maybe the mention of LazyDFU could be added as a recommendation there but I see no reason to remove this option entirely.

If it *is* kept, maybe a short explanation of exactly *why* it's dangerous might be important.

I don't know. I'll leave this open until someone else closes it so that other people can share their opinions too.
